### DISCUSSION TITLE
Official Figma Kit for Bootstrap

### DISCUSSION THREAD

**Discussion by jcusick93** [twbs/bootstrap#35707] [Ideas]
Is it possible to get an official Figma kit of Bootstrap?

**Comment by qvsoft** [comment]
Dear Bootstrap Team,
I love Bootstrap and have been a developer to build theme using Bootstrap for many years, since Bootstrap 2.
I did build a completely Design System on Figma for newest Bootstrap 5.1, that's completely **free** and follow **exactly** Bootstrap document and guideline, and continuous updating with fully examples and interactive prototype.
Check it out here: https://www.figma.com/community/file/1044316192441037087
I've experienced to build another DS which has 51k+ duplications on Figma, and the new one for Bootstrap 5 build from all my experiences. It will save designers hundreds hours of working to build a new one from scratch, follow Bootstrap guideline and help the design and the production code consistent, pixel perfect.
